bot/cogs/monstersurvey.py

```diff
@@ -0,0 +1,161 @@
+from discord.ext.commands import Bot, Context
+from asyncio import sleep as asleep
+from typing import Optional
+from discord.ext import commands
+from discord import Embed
+import logging
+import json
+import os
+
+log = logging.getLogger(__name__)
```

Comment:
Nothing is done with this, please add some appropriate logging statements to the rest of the cog.